source/image-handler/lambda_function.py

```diff
@@ -295,11 +295,11 @@ def request_thumbor(original_request, session):
     vary, request_headers = auto_webp(original_request, request_headers)
     return session.get(unix_path + http_path, headers=request_headers), vary
 
-def process_thumbor_responde(thumbor_response, vary):
+def process_thumbor_response(thumbor_response, vary):
      if thumbor_response.status_code != 200:
          return response_formater(status_code=thumbor_response.status_code)
      if vary:
-         vary = thumbor_response.headers['vary']
+         vary = 'Accept'
```

Comment:
@omrisiri We don't need to change anything here. If we forward the headers from cloudfront (https://aws.amazon.com/premiumsupport/knowledge-center/configure-cloudfront-to-forward-headers/ choose accept instead of host), this will work as it is. And I think we will have to forward the headers anyway since the thumbor needs to know the header value.

Reply:
@jainraj @omrisiri I've configured cloudfront to forward the `Accept` header, which works, but throws an error when you pass a format filter, eg: https://dlywxk1dfco56.cloudfront.net/Uz-eU0TnL0X_2ZbKHeCc1-lVyfs=/716x716/filters:focal(839x839:841x841):format(jpeg):quality(61):sharpen(0.5,0.5,true)/development.craftcms.stealthmonitoring/headshots/Rob-Cherun.jpg